### ISSUE TITLE
Open source ukui.org

### ISSUE THREAD

**Issue by Newbytee** [ukui/community#20]
I think it would be nice to have a repository for your website so that fixes can be contributed and the source code studied.

**Comment by handsome-feng** [comment]
Hi, @Newbytee , thanks for your suggestion, we'll consider it, have you found any errors so far?

**Comment by Newbytee** [comment]
> Hi, @Newbytee , thanks for your suggestion, we'll consider it, have you found any errors so far?

Yes, the same errors I corrected in https://github.com/ukui/ukui-desktop-environment/pull/1 are present on ukui.org, and I imagine the links shown in the image below should be vertically centred (they appear to have more bottom padding than top padding).
![image](https://user-images.githubusercontent.com/4428753/84494858-7f658980-acaa-11ea-86f0-54a36324c729.png)

In addition to this, visiting ukui.org directly (without going to www.ukui.org) presents me with what's shown in the image below (though, I imagine this is not something that could be fixed via code change):
![image](https://user-images.githubusercontent.com/4428753/84495065-d3706e00-acaa-11ea-8f7e-86611c33a26d.png)

**Comment by handsome-feng** [comment]
Thanks for your info, we will fix the warning of ukui.org recently and we are planning to redesign the entire site.
